zeebe/atomix/cluster/src/main/java/io/atomix/cluster/AtomixCluster.java

```diff
@@ -286,15 +297,23 @@ public String toString() {
   }
 
   /** Builds a default messaging service. */
-  protected static ManagedMessagingService buildMessagingService(final ClusterConfig config) {
+  protected static ManagedMessagingService buildMessagingService(
+      final ClusterConfig config, final String schedulerPrefix) {
     return new NettyMessagingService(
-        config.getClusterId(), config.getNodeConfig().getAddress(), config.getMessagingConfig());
+        config.getClusterId(),
+        config.getNodeConfig().getAddress(),
+        config.getMessagingConfig(),
+        schedulerPrefix);
```

Comment:
:x: Shouldn't we pass the whole scheduler name here instead of just prefix? So `schedulerPrefix`+memberId? `NettyMessagingService` do not have access to the memberId. So we have to build it here and pass it directly.

Reply:
We're doing that in the `AtomixClusterConfiguration` constructor.